plugins/CMakeLists.txt

```diff
@@ -49,6 +49,7 @@ IF("${PLUGIN_LIST}" STREQUAL "")
 		carlarack
 		CrossoverEQ
 		Delay
+		Disintegrator
```

Comment:
This part of `plugins/CMakeLists.txt` has been moved to `cmake/modules/PluginList.cmake` in 15fe551b1da9863db6b127665dd390473615513d